generation_config.yaml

```diff
@@ -1231,7 +1231,7 @@ libraries:
   - proto_path: google/cloud/managedidentities/v1beta1
 
 - api_shortname: managedkafka
-  name_pretty: Apache Kafka for BigQuery API
+  name_pretty: Managed Service for Apache Kafka API
```

Comment:
I believe this is the source of other lines.

Reply:
We shouldn't allow manual change in other files. I'll run the generation job for this library.